config/initializers/devise.rb

```diff
@@ -8,7 +8,7 @@
   # confirmation, reset password and unlock tokens in the database.
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
-  #config.secret_key = 'c9aac5e1283cb3302f0a7a2217dbd287a8b8564070b519d96246666ba172191d3b1267d1b578c0e1771890a15fe2ef30edbd3adc3cac38d6b9a2b918b8421ab9'
+  # config.secret_key = 'c9aac5e1283cb3302f0a7a2217dbd287a8b8564070b519d96246666ba172191d3b1267d1b578c0e1771890a15fe2ef30edbd3adc3cac38d6b9a2b918b8421ab9'
```

Comment:
I would leave this file as-is.  The comments style is designed to distinguish between commented out settings and docs about the settings.  Rubocop doesn't seem to grok this.